src/pages/tasks/TaskShareDestinationSelectorModal.js

```diff
@@ -158,25 +107,15 @@ function TaskShareDestinationSelectorModal(props) {
     );
 }
 
-TaskShareDestinationSelectorModal.displayName = 'TaskShareDestinationSelectorModal';
```

Comment:
Why do you remove display name? AFAIK, all functional components must have it.

Reply:
that was a mistake! thanks for that